pkg/gadget-context/gadget-context.go

```diff
@@ -189,6 +189,19 @@ func (c *GadgetContext) DataOperators() []operators.DataOperator {
 	return slices.Clone(c.dataOperators)
 }
 
+func (c *GadgetContext) IsRemoteCall() bool {
+	val := c.ctx.Value(remoteKey)
```

Comment:
We have some places where we can only forward the `context.Context` to - for example the ocistore or gRPC. By using this, the info is also carried down there. Not sure if we ever need it there, but it doesn't make a big difference, so I thought this would be nicer.

Reply:
Makes sense to me.